go/store/chunks/chunk.go

```diff
@@ -49,6 +49,10 @@ func (c Chunk) IsEmpty() bool {
 	return len(c.data) == 0
 }
 
+func (c Chunk) Size() int {
```

Comment:
Lets kill this change along with the datas/pull.go changes as they are no longer applicable in the latest and will just cause conflicts.